src/main/java/org/reaktivity/nukleus/tcp/internal/stream/ClientStreamFactory.java

```diff
@@ -196,9 +193,13 @@ public void doConnect(
         SocketChannel channel,
         InetSocketAddress remoteAddress,
         String acceptReplyName,
-        long correlationId)
+        long correlationId,
+        MessageConsumer outputThrottle,
+        long outputStreamId,
+        BiConsumer<MessageConsumer, Long> setCorrelatedInput)
```

Comment:
Define an `ObjectLongBiConsumer` functional interface to avoid auto-boxing from `long` to `Long` at runtime - can produce quite a bit garbage.